bigquery/.flake8

```diff
@@ -3,6 +3,7 @@ exclude =
   # Exclude generated code.
   **/proto/**
   **/gapic/**
+  **/types.py
```

Comment:
Not relevant for BigQuery, and might actually clash with a manually-created file later.

Reply:
@tseaver, I know we are planning to use templating later. I can remove this for now, but I expect that later it will get added back in.  Though, to be fair, I also expect `black`ening to happen soonish also.

Reply:
@crwilcox My point is that Bigquery is a manual, not-even-gRPC wrapper, and that masking `types.py` from is both pointless (we don't have an autogenned `types.py` now, and likely never will) and creates a potential future hygeine bug (should we ever create `types.py` by hand).

Blindly applying templates to repositories without considering downsides / fitness-for-purpose is not going to be a win.